configure.ac

```diff
@@ -52,6 +52,7 @@ FUZZ_CPPFLAGS="${FUZZ_CPPFLAGS:--DASS_FUZZMODE=0}"
 
 # Checks for flags support
 AX_APPEND_COMPILE_FLAGS([-fno-math-errno])
+AX_APPEND_COMPILE_FLAGS([-clang:-fno-math-errno])
```

Comment:
nitpick, but `clang-cl` uses nominally cl.exe style arguments, so for consistency it should be `/clang:-fno-math-errno`. I believe both should work, but just point this out.